etl/models/arkg_schema.py

```diff
@@ -0,0 +1,11 @@
+from pyoxigraph import NamedNode
+
+
+class ArkgSchema:
```

Comment:
I usually model namespaces like:

```
namespaces/arkg.py

class Arkg:
   X = NamedNode(...)
```

```
namespaces/rdf.py

class RDF:
   TYPE = ...
```

Then use `namespaces/__init__.py` to import them as RDF, ARKG

rdflib, Jena, and other libraries use a similar design (e.g., https://github.com/tpluscode/rdf-ns-builders/tree/master/packages/lib).